docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV PRESTO_HOME="/opt/presto-server"
 COPY $PRESTO_PKG .
 COPY $PRESTO_CLI_JAR /opt/presto-cli
 
-RUN dnf install -y java-11-openjdk less procps python3 \
+RUN dnf install -y java-17-openjdk less procps python3 \
```

Comment:
I tested this out locally and the RedHat Open JDK 17 installed will have TZ data updated to the latest version. This will break some of the TZ test cases. Should we instead change this to manually use a [Temurin 17.0.13](https://github.com/adoptium/temurin17-binaries/releases/tag/jdk-17.0.13%2B11) OpenJDK pinned to a specifc version ?